cinder/volume/drivers/nexenta/utils.py

```diff
@@ -42,7 +42,7 @@ def str2size(s, scale=1024):
 
     groups = match.groups()
     value = float(groups[0])
-    suffix = len(groups) > 1 and groups[1].upper() or 'B'
+    suffix = groups[1].upper() if len(groups) else 'B'
```

Comment:
What if len(groups) == 1?

Reply:
len(groups) can never be 1 because of the regex used in re.match() (line 39).
match will either be None or match.groups() will return a list with len == 2.

Reply:
Yeah, I had not checked regexp. I just spotted that the condition changed. 
In this case, why to check length of groups if it always equals 2?
Maybe it should be:
suffix = groups[1].upper() if len(groups[1]) else 'B'
?

Reply:
it should actually just be 
suffix = groups[1].upper() if groups[1] else 'B'
this wasn't my code and I missed it, thank you!